csrc/expr_evaluator.cpp

```diff
@@ -232,4 +232,17 @@ void ExpressionEvaluator::propagateBoundValuesThroughExactMaps(Fusion* fusion) {
   }
 }
 
+ExpressionEvaluator ExpressionEvaluator::clone(IrCloner& ir_cloner) const {
+  ExpressionEvaluator expr_eval;
+  TORCH_INTERNAL_ASSERT(
+      !precomputed_values_,
+      "Cannot clone ExpressionEvaluator with bound PrecomputedValues");
+  for (const auto& kv : known_values_) {
+    expr_eval.known_values_[ir_cloner.clone(kv.first)] = kv.second;
+  }
+  expr_eval.known_named_scalars_.insert(
+      known_named_scalars_.begin(), known_named_scalars_.end());
+  return expr_eval;
+}
+
```

Comment:
These are needed in order to clone the partially-evaluated expr_eval to the new `Fusion` which will be concretized.